code/game/machinery/gulag_teleporter.dm

```diff
@@ -0,0 +1,129 @@
+/*
+The gulag teleporter teleports a prisoner to the gulag outpost.
+It automatically strips the prisoner and equips a prisoner ID, prisoner jumpsuit and oranges sneakers.
+You can set the amount of points in the console
+The console is located at computer/gulag_teleporter.dm
+*/
+
+//Gulag teleporter
+/obj/machinery/gulag_teleporter
+	name = "labor camp teleporter"
+	desc = "A bluespace teleporter used for teleporting prisoners to the labor camp."
+	icon = 'icons/obj/machines/implantchair.dmi'
+	icon_state = "implantchair"
+	state_open = 0
+	density = 1
+	anchored = 1
+	use_power = 1
+	idle_power_usage = 200
+	active_power_usage = 5000
+	var/locked = 0
+	var/list/stored_items = list()
+	var/jumpsuit_type = /obj/item/clothing/under/rank/prisoner
+	var/shoes_type = /obj/item/clothing/shoes/sneakers/orange
+
+/obj/machinery/gulag_teleporter/interact(mob/user)
+	if(locked)
+		user << "[src] is locked."
+		return
+	toggle_open()
+
+/obj/machinery/gulag_teleporter/updateUsrDialog()
+	return
+
+/obj/machinery/gulag_teleporter/update_icon()
+	//no power or maintenance -- no sprites currently
+/*	if(stat & (NOPOWER|BROKEN))
+		icon_state = initial(icon_state) + (state_open ? "_open" : "") + "_unpowered"
+		return
+
+	if((stat & MAINT) || panel_open)
+		icon_state = initial(icon_state) + (state_open ? "_open" : "") + "_maintenance"
+		return */
+
+	//running and someone in there
+	if(occupant)
+		icon_state = initial(icon_state) + "_occupied"
+		return
+
+	//open/closed with no occupant
+	icon_state = initial(icon_state) + (state_open ? "_open" : "")
+
+/obj/machinery/gulag_teleporter/relaymove(mob/user as mob)
+	if(user.stat != CONSCIOUS)
+		return
+	if(locked)
+		user << "[src] is locked!"
+	open_machine()
+
+/obj/machinery/gulag_teleporter/container_resist()
+	var/mob/living/user = usr
+	var/breakout_time = 1
+	if(!locked)
+		open_machine()
+		return
+	user.changeNext_move(CLICK_CD_BREAKOUT)
+	user.last_special = world.time + CLICK_CD_BREAKOUT
+	user << "<span class='notice'>You lean on the back of [src] and start pushing the door open... (this will take about [breakout_time] minutes.)</span>"
+	user.visible_message("<span class='italics'>You hear a metallic creaking from [src]!</span>")
+
+	if(do_after(user,(breakout_time*60*10), target = src)) //minutes * 60seconds * 10deciseconds
```

Comment:
it's more to show the message to the user in minutes(i can change to keep it cleaner if that's a problem)